client/galaxy/scripts/mvc/library/library-foldertoolbar-view.js

```diff
@@ -205,46 +205,38 @@ var FolderToolbarView = Backbone.View.extend({
       return folderDetails.name !== '';
   },
 
-
   // show bulk import modal
   modalBulkImport : function(){
       var checkedValues = $('#folder_table').find(':checked');
       if(checkedValues.length === 0){
           mod_toastr.info('You must select some datasets first.');
       } else {
-          this.refreshUserHistoriesList(function(that){
-            var template = that.templateBulkImportInModal();
-            that.modal = Galaxy.modal;
-            that.modal.show({
-                closing_events  : true,
-                title           : 'Import into History',
-                body            : template({histories : that.histories.models}),
-                buttons         : {
-                    'Import'    : function() {that.importAllIntoHistory();},
-                    'Close'     : function() {Galaxy.modal.hide();}
-                }
-            });
+        var that = this;
```

Comment:
We have two approaches to referring to instantiated 'this' in Javascript, and I think for the most part, we have standardized on using 'self'.  Would you mind using the same nomenclature here?

Reply:
Changing it at one place seems pointless. Changing it everywhere seems like a project I do not want to do.

```
/client$ ack -hc 'var that'
45
```

Reply:
I checked the count before commenting.  437 'this', so 10x more commonly used in the codebase.  Like any other refactoring, making small improvements as we rewrite code is a nice way to progress.  I won't -1, of course, if you'd rather not.  Someone's going to have to do it though.

Reply:
I politely disagree, this does not have to be done.

Reply:
Okay.

Reply:
Soon we will have arrow functions and that should kill a lot of these. Not worth worrying about in the meantime.

(put my comment in the wrong place)

Reply:
We've made a lot of steps towards code style standards lately, and this was a suggestion of an incremental step towards standards that we use across the codebase.  This file, in particular, uses *both* paradigms from method to method, and it'd certainly be easier to read if it just picked one -- I just have to say I'm pretty surprised by the pushback here.

That we'll have arrow functions soon is a good point, and will definitely help as we're able to update all these, but that's a *much* larger project that I'd be blown away if we accomplish across much of the javascript codebase in the next few months.  That project would also arguably benefit from us being able to quickly grep for instance methods using a standard approach.

Anyway, wasn't looking for an argument here, just a minor suggestion.  Merging anyway.

Reply:
@martenson I'm not going to hold up future PRs or anything - I find these JS PRs particularly problematic to revisit because of the packed script issue.  But I do find the JavaScript quite overwhelming and a part of that is varying spacing and naming conventions across the codebase. Can we agree we don't need to revisit things for this PR but for future code use ``self`` because that is more common? I suspect I've used both in the past depending on where I was - but it would be good to agree to stick with on IMO - it would help me be more confident about what I was reading and writing I think.

Reply:
@jmchilton @dannon Will do. The spacing problem should go away with the webpack upgrade I expect.